modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -106,6 +106,27 @@ Your username must be at least three characters and contain only the following:
 [NOTE]
 You cannot create a new account within an existing organization without an email invitation.
 
+== Sign in to Anypoint Platform
+
+. Navigate to the Anypoint Platform signin screen.
+. Enter your username and password.
+. Click *Sign in*.
+
+[NOTE]
+If you try to sign in and Anypoint Platform says your username and password are incorrect when you have entered the correct credentials, your organization may use an xref:access-management::external-identity.adoc[external identity provider] to initiate the sign-in process. You can either navigate to your external identity provider's interface to sign in, or you can xref:access-management::managing-your-account.adoc#use-CustomDomainLogin[enter your organization's domain name] prior to entering your credentials. 
+
+[[use-CustomDomainLogin]]
+== Sign in to Anypoint Platform using Single Sign-on
+
+If your organization uses xref:access-management::external-identity.adoc[Single Sign-on], you must provide your organization's domain name as part of the sign-in process.
+
+. Navigate to the Anypoint Platform signin screen.
+. Click *Use custom domain*.
+. Enter your organization's domain name. If you do not know your organization's domain name, a member of your organization can provide the organization domain name, which is accessible in their user profile.
```

Comment:
GLOBAL step or list-item format=max 1 sentence:
```suggestion
. Enter your organization's domain name. 
+
+ If you do not know your organization's domain name, you can find it in your user profile or ask another member of your organization for it.
```
? Why wouldn't the user not have it in their own user file?

Reply:
Because they haven't logged in yet.

Reply:
```suggestion
. Enter your organization's domain name.
+
+ If you do not know your organization's domain name, a member of your organization can provide the organization domain name, which is accessible in their user profile.
```